stages/org.osbuild.pacman.conf

```diff
@@ -15,24 +15,82 @@ SCHEMA = """
     "type": "string",
     "description": "pacman architecture",
     "default": "x86_64"
+  },
+  "mirrors": {
+    "type": "array",
+    "items": {
+      "type": "string"
+    }
+  },
```

Comment:
This now hardcodes a default location

Reply:
It's not an issue if it just uses the default location, but you can add it with the default as fallback if you want.